Validation/src/Validation/simparticles.py

```diff
@@ -22,6 +22,14 @@ def beamenergy_comp(d: Differ, out_dir=None):
              tick_labels=pdgid_labels,
              out_dir=out_dir,
              density=True)
+    d.plot1d("SampleValidation/SampleValidation_energy_daughterphoton", "Energy spectrum of all photons from primary",
+             out_dir=out_dir,
+             density=True)
+
+    d.plot1d("SampleValidation/SampleValidation_pdgid_harddaughters" "PDG ID of hard primary daughter",
+             tick_labels=pdgid_labels,
```

Comment:
This is really great. Makes it SO much easier to read and grasp the plot!

Reply:
+1